src/physics/em/detail/PhysicsConstants.hh

```diff
@@ -59,6 +59,22 @@ CELER_CONSTEXPR_FUNCTION MevPerCm lpm_constant()
 }
 //!@}
 
+//!@{
+//! Constant functions for model limits
+
+//! Maximum energy for the SeltzerBerger model to be valid
+CELER_CONSTEXPR_FUNCTION units::MevEnergy seltzer_berger_limit()
+{
+    return units::MevEnergy{1e3}; //! 1 GeV
+}
+
+//! Maximum energy for EM models to be valid
+CELER_CONSTEXPR_FUNCTION units::MevEnergy high_energy_limit()
```

Comment:
Since this is in a `celeritas::detail` namespace rather than `celeritas::detail::brems` or something, maybe rename it `rel_brems_limit`?

Reply:
Actually, this high_energy_limit should be a common default value for all EM models, except for the SB model.  So, I would like to keep this as it is, but add a specific limit if there is any variation later.